Stylistic changes from Spark maintainers

ajacques opened the pull request:
Improvements as requested by Spark maintainers in apache/spark#21889

mallman commented:
Thanks for your PR, @ajacques. I will review in earnest.

mallman commented:
@ajacques Thanks for addressing the merge conflicts. I've been occupied with some other work-related responsibilities today. I'll try to get to this tomorrow (Friday).

Cheers.

mallman commented:
@ajacques Should I give co-attribution to @HyukjinKwon for this work? As-is it only has your name on it.

HyukjinKwon commented:
I'm okay in any way.

mallman commented:
@ajacques I removed my changes to `ParquetTest.scala` upstream. Can you do a rebase and verify that your PR includes no changes to `ParquetTest.scala`? Thanks.

mallman commented on sql/core/src/test/scala/org/apache/spark/sql/execution/SelectedFieldSuite.scala:
```diff
@@ -27,309 +27,373 @@ import org.apache.spark.sql.catalyst.parser.CatalystSqlParser
-  info("For a relation with schema\n" + indent(schema.treeString))
+  //  |-- col1: string (nullable = false)
+  //  |-- col2: struct (nullable = true)
+  //  |    |-- field1: integer (nullable = true)
```
I don't see `field1` in `structOfArray`. Please remove it here or add it there.

ajacques replied:
You're correct, good catch. I've removed this one. `field1` is belongs in the previous schema.

ajacques commented:
> @ajacques Should I give co-attribution to @HyukjinKwon for this work? As-is it only has your name on it.

Sure, let's do that. HyukjinKwon's comments were a valuable resource in this pull request. Is there anything I need to do on my end for this?

> @ajacques I removed my changes to ParquetTest.scala upstream. Can you do a rebase and verify that your PR includes no changes to ParquetTest.scala? Thanks.

Rebased, and checked that there's no changes to that file now.

mallman commented:
Merged. Thank you.
